app/views/completed_transaction/_register_to_vote_promotion.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="register-to-vote-promotion"
+     data-module="auto-track-event"
+     data-track-category="linkTrack"
+     data-track-action="linkDisplayed"
+     data-track-label="Register">
+  <p>You must <a href="/register-to-vote">register to vote</a> by 7 June if you want to take part in the EU referendum. You can register online and it only takes 5 minutes.</p>
```

Comment:
Just gonna drop this commit; now there's only one promo I don't think we gain anything from partialising it.  Good spot, thanks for the eagle-eye!